test/unit/cmap/connect.test.ts

```diff
@@ -242,6 +242,7 @@ describe('Connect Tests', function () {
           };
           const handshakeDocument = await prepareHandshakeDocument(authContext);
           expect(handshakeDocument).not.to.have.property('loadBalanced');
+          expect(handshakeDocument).to.have.property(LEGACY_HELLO_COMMAND, 1);
```

Comment:
I think in this case, to surface that it is an explicit requirement that we have `LEGACY_HELLO_COMMAND` set to 1, we should break this assertion out into its own `it` block with the set up of the handshake document done in a `beforeEach`. That lets us have a test case in our output that says something like

```
when loadBalanced is not set as an option
	does not set loadBalanced on the handshake document ✅ 
	sets LEGACY_HELLO_COMMAND to 1			    ✅ 	
```